src/libraries/Common/src/Interop/Windows/WinSock/Interop.WSADuplicateSocket.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Net.Sockets;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Winsock
+    {
+        [StructLayout(LayoutKind.Sequential, CharSet=CharSet.Auto)]
```

Comment:
I think we can get rid of this attribute; sequential is the default and there are no characters to marshal here.

Reply:
Isn't `LayoutKind.Auto` the default? All pinvoke code I've ever seen keeps adding `[StructLayout(LayoutKind.Sequential)]`, I thought this ancient rite is being done for a reason 😄

`CharSet` is indeed meaningless.

Reply:
Structs use sequential layout by default in c#. It's probably being added just to make explicit that it will be used for pinvoke -- fine to keep for consistency.